infra/templates/dns-records/template.yaml

```diff
@@ -42,36 +46,80 @@ Resources:
             - !Ref Environment
             - DNSSUFFIX
       SubjectAlternativeNames:
-        - !Sub
-          - api.${DNSSUFFIX}
-          - DNSSUFFIX: !FindInMap
-              - PlatformConfiguration
-              - !Ref Environment
-              - DNSSUFFIX
         - !If
-          - IsDevEnvironment
+          - IsBuildOrStagingEnvironment
           - !Sub
-            - '*.${DNSSUFFIX}'
+            - sessions-mob-async-backend.${DNSSUFFIX}
             - DNSSUFFIX: !FindInMap
                 - PlatformConfiguration
                 - !Ref Environment
                 - DNSSUFFIX
           - !Ref AWS::NoValue
-      DomainValidationOptions:
-        - DomainName: !Sub
-            - ${DNSSUFFIX}
+        - !If
+          - IsBuildOrStagingEnvironment
+          - !Sub
+            - mob-sts-mock.${DNSSUFFIX}
```

Comment:
Please may you _double check_ the DNS for this service in the output of the sts-mock stack? I have a suspicion it may have changed recently